evm_loader/test_cli_emulate.py

```diff
@@ -139,96 +165,250 @@ def test_successful_cli_emulate(self):
+                {
+                    "pubkey": '' + self.token_acc1,
+                    "is_signer": False,
+                    "is_writable": True,
+                },
+                {
+                    "pubkey": '' + self.token_acc2,
+                    "is_signer": False,
+                    "is_writable": True,
+                },
+                {
+                    "pubkey": str(self.acc.public_key()),
+                    "is_signer": True,
+                    "is_writable": False,
+                },
+            ],
             "accounts": [
                 {
-                    "account": "CONTRACT_ACCOUNT",
-                    "address": "CONTRACT_ETHEREUM_ID",
-                    "code_size": null,
-                    "contract": "CONTRACT_CODE_ACCOUNT",
-                    "new": false,
-                    "writable": true
+                    "account": '' + self.contract.contract_account,
+                    "address": '0x' + self.contract.ethereum_id.hex(),
+                    "code_size": None,
+                    "contract": '' + self.contract.contract_code_account,
+                    "new": False,
+                    "writable": True,
                 },
                 {
-                    "account": "ETHEREUM_CALLER_ACCOUNT",
-                    "address": "ETHEREUM_CALLER_ID",
-                    "code_size": null,
-                    "contract": null,
-                    "new": false,
-                    "writable": true
-                }
+                    "account": '' + self.caller,
+                    "address": '0x' + self.ethereum_caller.hex(),
+                    "code_size": None,
+                    "contract": None,
+                    "new": False,
+                    "writable": True,
+                },
             ],
             "exit_status": "succeed",
             "result": ""
-        }"""
-        tmpl = tmpl.replace('CONTRACT_ACCOUNT', self.contract.contract_account)
-        tmpl = tmpl.replace('CONTRACT_ETHEREUM_ID', '0x' + self.contract.ethereum_id.hex())
-        tmpl = tmpl.replace('CONTRACT_CODE_ACCOUNT', self.contract.contract_code_account)
-        tmpl = tmpl.replace('ETHEREUM_CALLER_ACCOUNT', self.caller)
-        tmpl = tmpl.replace('ETHEREUM_CALLER_ID', '0x' + self.ethereum_caller.hex())
-        tmpl_json = json.loads(tmpl)
+        }
 
-        self.compare_tmpl_and_emulate_result(tmpl_json, emulate_result)
+        trx_data = abi.function_signature_to_4byte_selector('transferExt(uint256,uint256,uint256,uint256,uint256)') \
+                   + bytes.fromhex(base58.b58decode(self.token).hex()
+                                   + base58.b58decode(self.token_acc1).hex()
+                                   + base58.b58decode(self.token_acc2).hex()
+                                   + "%064x" % (transfer_amount * (10 ** 9))
+                                   + bytes(self.acc.public_key()).hex()
+                                   )
```

Comment:
What difference between these lines and https://github.com/neonlabsorg/neon-evm/blob/1693c363d125db9716f4194410bafda52e0d2adc/evm_loader/test_cli_emulate.py#L35-L41?

Reply:
It will refactor in #74 (a new function will be introduced)